### PR TITLE
Keep EXIF rotation during YOLO importing

### PR DESCRIPTION
**Pull request by sizov-kirill**
### Checklist

- [ ] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/cvat-ai/datumaro/blob/develop/CHANGELOG.md)
- [ ] I have updated the [documentation](https://github.com/cvat-ai/datumaro/tree/develop/docs) accordingly
- [x] I have added tests to cover my changes
- [ ] I have [linked related issues](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/cvat-ai/datumaro/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2022 CVAT.ai Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

**Comment by sizov-kirill** [comment]
@zhiltsov-max Please review this PR. The main changes are in `util/image.py` and `plugins/yolo_format/extractor.py`. Anything else was made just to make this PR green

File: datumaro/plugins/yolo_format/extractor.py [hunk 1]
```diff
@@ -156,14 +162,23 @@ def name_from_path(cls, path: str) -> str:
 
         return osp.splitext(path)[0]
 
+    @classmethod
+    def _image_loader(cls, *args, **kwargs):
+        return load_image(*args, **kwargs, keep_exif=True)
```
Comment (zhiltsov-max) [inline datumaro/plugins/yolo_format/extractor.py:167]: I like that this change is local for yolo, but at the same time other formats are probably suffering from this as well.

Comment (sizov-kirill) [reply datumaro/plugins/yolo_format/extractor.py:167]: Yes, I would suggest merging this hotfix first, and then implement a more universal solution

**Comment by sizov-kirill** [comment]
@zhiltsov-max I've added test

**Review by zhiltsov-max — APPROVED** [review]

**Pull request merged**